cmd/karavictl/cmd/inject.go

```diff
@@ -186,15 +200,21 @@ func buildProxyContainer(ns, configName, imageAddr, proxyHost string, insecure b
 	return &proxyContainer
 }
 
+const (
+	DefaultStartingPortRange = 9000
```

Comment:
Exported constant will need a comment.

Reply:
Should be fixed by:

https://github.com/dell/karavi-authorization/pull/86/commits/bd9df2a676bcd24109d79c81e6a3537938ce2780